ci/pipelines/deploy-to-production.yml

```diff
@@ -346,6 +356,60 @@ groups:
       - update-deploy-to-prod-pipeline
 
 jobs:
+  - name: deploy-carbon-relay-to-prod
+    plan:
+      - get: carbon-relay-ecr-registry-prod
+        trigger: true
+      - get: pay-infra
+      - get: pay-ci
+      - load_var: carbon_relay_image_tag
+        file: carbon-relay-ecr-registry-prod/tag
+      - task: create-notification-snippets
+        file: pay-ci/ci/tasks/create-carbon-relay-notification-snippets.yml
+        params:
+          CARBON_RELAY_IMAGE_TAG: ((.:carbon_relay_image_tag))
+          ENV: production-2
+      - load_var: failure_snippet
+        file: snippet/failure
+      - load_var: success_snippet
+        file: snippet/success
+      - load_var: start_snippet
+        file: snippet/start    
+      - <<: *put_start_slack_notification  
+      - task: assume-role
+        file: pay-ci/ci/tasks/assume-role.yml
+        params:
+          AWS_ROLE_ARN: arn:aws:iam::((pay_aws_prod_account_id)):role/concourse
+          AWS_ROLE_SESSION_NAME: terraform-prod-assume-role
+      - load_var: role
+        file: assume-role/assume-role.json
+        format: json
+      - task: check-release-versions
+        file: pay-ci/ci/tasks/check-release-versions.yml
+        params:
+          <<: *aws_assumed_role_creds
+          AWS_REGION: "eu-west-1"
+          CLUSTER_NAME: "production-2-fargate"
+          APP_NAME: carbon-relay
+          CARBON_RELAY_IMAGE_TAG: ((.:carbon_relay_image_tag))
+      - task: deploy-to-prod
+        file: pay-ci/ci/tasks/deploy-carbon-relay.yml
+        params:
+          <<: *aws_assumed_role_creds
+          CARBON_RELAY_IMAGE_TAG: ((.:carbon_relay_image_tag))
+          STUNNEL_IMAGE_TAG: test
```

Comment:
So this would be `0-release` at the moment for prod

Reply:
I noticed I hadn't committed it at the same time you commented! Pushed now.